modules/scripts_postprocessing.py

```diff
@@ -124,7 +124,7 @@ def run(self, pp: PostprocessedImage, args):
             script_args = args[script.args_from:script.args_to]
 
             process_args = {}
-            for (name, component), value in zip(script.controls.items(), script_args):
+            for (name, component), value in zip(script.controls.items(), script_args):  # noqa B007
                 process_args[name] = value

```

Comment:
You can prefix the unused variable with an underscore too, but I guess this would simplify to

```suggestion
            process_args = {name: value for (name, _component), value in zip(script.controls.items(), script_args)}
```